### DISCUSSION TITLE
ccf-validation-tools/Main_Bronchus/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#276] [General]
# ccf-validation-tools/Main_Bronchus/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Main_Bronchus/comments/

**Comment by emquardokus** [comment]
main-bronchus-v1.1_DRAFT
https://docs.google.com/spreadsheets/d/13O5Bb5_ki6uj4NcrFYoZPCHbYwrsPcyPJtdamUeZWZA/edit#gid=0

**Blank ontology ID missing parent**
-Replaced ciliated cell of terminal ciliated ducts of submucosal glands	LMHA:00142 with airway submucosal gland duct ciliated cell CL:4033055; @aleixpuigb is there a separate request for "terminal ciliated ducts" version of this cell type?    In row 15, the term ciliated cell of terminal ciliated ducts of submucosal glands without ontology ID has no parent that is from the CL ontology.
-ADDED respiratory suprabasal cell	CL:4033048   In row 25, the term respiratory suprabasal cell without ontology ID has no parent that is from the CL ontology.
-ADDED deuterosomal cell	CL:4033044 since the definition and position in CL is now under respiratory epithelial cell    In row 28, the term airway deuterosomal cell without ontology ID has no parent that is from the CL ontology.

**Reply by gpryhuber** [reply]
Agree to use  airway submucosal gland duct ciliated cell CL:4033055 in the main bronchus table ("terminal ciliated ducts" not needed at this point
Addition of CL:4033048 is correct in row 25
Addition of CL:4033044 is also correct in row 28

**Reply by aleixpuigb** [reply]
>  @aleixpuigb is there a separate request for "terminal ciliated ducts" version of this cell type?

There isn't an open ticket/request for this term.

**Comment by emquardokus** [comment]
main-bronchus-v1.1
https://docs.google.com/spreadsheets/d/13O5Bb5_ki6uj4NcrFYoZPCHbYwrsPcyPJtdamUeZWZA/edit#gid=0

**Different labels**

ADDED CL label:     The term CL:4033023 has a different name/label in the source ontology in the following 1 row 16. The name/label in the ASCT+B table is __ and the one in the ontology is airway submucosal gland collecting duct epithelial cell. For reference, the given name/label by SMEs is airway submucosal gland collecting duct epithelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
**main-bronchus-v1.2 https://docs.google.com/spreadsheets/d/1zAvpQa2or2uVKf7_oK7_kSWMFIs83mdPIO5tb-FwYnA/edit#gid=0**

**Different label**
ADD the ontology label to blank cell to correct: The term CL:4033023 has a different name/label in the source ontology in the following 1 row 16. The name/label in the ASCT+B table is __ and the one in the ontology is airway submucosal gland collecting duct epithelial cell. For reference, the given name/label by SMEs is airway submucosal gland collecting duct epithelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.
